### ISSUE TITLE
Check whitespace after redirection operators

### ISSUE THREAD

**Issue by szepeviktor** [koalaman/shellcheck#1344]
#### For new checks and feature suggestions
- [x] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [x] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh

#!/bin/bash

ipset -exist restore < whitespace.ipset
```

#### Here's what shellcheck currently says:

""

#### Here's what I wanted or expected to see:

Please avoid using a space after redirection operators.

Bash help also avoids it: `[n]>word`

**Comment by koalaman** [comment]
Having a space after a redirection operator is valid and causes no problems, so it's a subjective stylistic choice that ShellCheck doesn't aim to get involved in. I would argue that having a space improves readability, and also helps with e.g. `< <(cmd)` where a space is in fact required.

**Comment by szepeviktor** [comment]
Okay.
I've seen it in [`man bash`](https://manpages.debian.org/stretch/bash/bash.1.en.html)
